### PR TITLE
[Live] Ensure we don't show an emprty descripton, or artist blurb

### PR DESCRIPTION
**Pull request by orta**
. And don't show links in markdown.

Still not 100% convinced that `artwork.blurb` is what we want for the description.

### CODE REVIEW

File: Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift [hunk 1]
```diff
@@ -65,13 +65,13 @@ class AuctionLotMetadataStackScrollView: ORStackScrollView {
 
         let separatorMargin = String(Int(sideMargin) ?? 0 + 40)
 
-        if let description = viewModel.lotArtworkDescription {
+        if let artworkdescription = viewModel.lotArtworkDescription where artworkdescription.isEmpty == false {
```
Comment (ashfurrow) [inline Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift]: Well done 👏

**Comment by ashfurrow** [comment]
Looks 💯

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
        
            1 Error
        
      </th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">There were <a href="https://eigen-ci.s3.amazonaws.com/2685/index.html">snapshot errors</a></td>
    </tr>
    
    
  </tbody>
</table>
  

  
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
        
            1 Warning
        
      </th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
    
    
  </tbody>
</table>
  

  



Time | Class | Function |
| --- | ----- | ----- |237.3ms | [AuctionViewController.swift](/artsy/eigen/tree/orta-artwork_metadata/Artsy/View_Controllers/Auction/AuctionViewController.swift#L181) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
204.3ms | [SwiftyJSON.swift](/artsy/eigen/tree/orta-artwork_metadata/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
194.8ms | [WebSocket.swift](/artsy/eigen/tree/orta-artwork_metadata/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
80.4ms | [Observable.swift](/artsy/eigen/tree/orta-artwork_metadata/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
67.6ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/orta-artwork_metadata/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, sideMargin: String)
61.2ms | [AuctionBannerView.swift](/artsy/eigen/tree/orta-artwork_metadata/Artsy/Views/Auction/AuctionBannerView.swift#L41) | private func setupViews()
59.8ms | [RefinementOptionsViewController+Private.swift](/artsy/eigen/tree/orta-artwork_metadata/Artsy/View_Controllers/Auction/RefinementOptionsViewController+Private.swift#L63) | func stackView() -> ORStackView




<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by ashfurrow** [comment]
Snapshot needs a swap.

**Pull request merged**